README.md

```diff
@@ -12,12 +12,31 @@ A list of awesome [RemNote](https://remnote.io) plugins, themes, tutorials and o
  such that the devs can focus on RemNote itself. The builtin tutorials
  are great but there are some outdated and missing parts.
  -->
-
+ <!-- Add contributing guidelines -->
+- [Awesome Remnote](#awesome-remnote)
+  - [Community](#community)
+  - [Integrations](#integrations)
+  - [Extensions](#extensions)
+    - [Plugins](#plugins)
+    - [Themes](#themes)
+    - [Custom CSS Mixins](#custom-css-mixins)
+    - [Development](#development)
+  - [Tutorials](#tutorials)
+    - [Video Tutorials](#video-tutorials)
+    - [Courses](#courses)
+- [Education Materials](#education-materials)
+  - [Learning](#learning)
+    - [Spaced Repetition](#spaced-repetition)
+    - [Note-taking](#note-taking)
+  - [Knowledge Bases](#knowledge-bases)
+
+
+# Awesome Remnote
```

Comment:
I intended the banner above to be the top level heading. I would remove the heading again. What do you think?

Reply:
I think it may be prudent to leave the top part as is so people who stumble onto this repo know what it's even about, then go into the awesome list part of it, but it's up to you.

Reply:
Hmm, good point. But maybe the repo name + the intro text (A curated list of RemNote plugins, themes, tutorials and other resources.) which is also displayed on the social image card (which people mainly share I think) is enough?

Reply:
To be honest, at this point it's a matter of your taste. Maybe when (if) we hammer out a contribution guide it'd be worth spending time on cosmetics. As long as the TOC is readable and easy to link to, I'm happy.